R/summarize.R

```diff
@@ -1,29 +1,85 @@
 #' @export
-#' @importFrom dplyr summarise_
+#' @importFrom dplyr bind_cols summarise_ select
+#' @importFrom purrr map_chr map_df
 #' @importFrom crunch crtabs
 summarise_.CrunchDataset <- function (.data, ..., .dots) {
     dots <- all_dots(.dots, ..., all_named = TRUE)
-    fmla <- dots_to_formula(dots, groups(.data))
-    out <- as_tibble(crtabs(fmla, data=.data))
-    ## Sort the return columns based on the request order
-    out <- out[, c(setdiff(names(out), names(dots)), names(dots))]
+    unweighted <- dots %>% map_chr(~as.character(.$expr)[[1]]) == "unweighted_n"
+    counts <- dots[unweighted]
+    non_counts <- dots[!unweighted]
+    fmla <- dots_to_formula(non_counts, groups(.data))
```

Comment:
I know that it means formula here, but I cannot stop reading this as the [Family and Medical Leave act](https://en.wikipedia.org/wiki/Family_and_Medical_Leave_Act_of_1993)

Reply:
That's the correct reading.